content/use_cases/use_cases.adoc

```diff
@@ -0,0 +1,210 @@
+|===
+
+[cols="1,5",caption='Use case 10: ']
+.Simulating energy/signal propagation with 3D assets linked to material properties
+|===
+|*Type of use case*         |Technical use case
+|*Description*              |As a simulation platform or sensor model developer, I want to simulate the energy/signal propagation using imported 3D assets with linked material properties. This is, for example, done with ray tracing. Rays are launched in a virtual 3D scene to simulate the propagation of light beams, radio waves, or ultrasonic waves. The interaction of the rays with the surfaces of the objects in the 3D environment depends on the material properties of these surfaces. These properties are assigned to the 3D geometries of the objects and imported from a material database.
+
+The simulation shall be able to cope with different real-time requirements, for example, SiL, HiL, open-loop, closed-loop etc.
+|*Material and 3D Structure*|Materials and 3D Structure
+|*Influencing Standards*    |
+|*Influencing Technologies* |Nvidia OptiX
+|*Used 3D Model Formats*    |glTF, FBX, USD, OSG, OBJ
+|*Used Material Formats*    |
+|*Companies*                |ADaC, Ansys, Persival, PilotD
+|*Relevant User*            |Simulation platform developer, sensor model developer, end user
+|===
+
+[cols="1,5",caption='Use case 11: ']
+.Using sensor simulation to train perception algorithms
+|===
+|*Type of use case*         |Technical use case
+|*Description*              |As a perception algorithm developer, I want to use simulated environments for model training and testing, as real-world information collection is too expensive and inconvenient.
+|*Material and 3D Structure*|Materials and 3D Structure
+|*Influencing Standards*    |ASAM OSI, ASAM OpenSCENARIO, ASAM OpenDRIVE
+|*Influencing Technologies* |Unity, OptiX, Regeneration AI
+|*Used 3D Model Formats*    |
+|*Used Material Formats*    |
+|*Companies*                |Jiao Tong University, ViF
+|*Relevant User*            |End user
+|===
+
+////
+[cols="1,5",caption='Use case 12: ']
+.Sharing and reusing 3D models and material data in different simulation platforms and sensor models with varying sensor technology (wavelength)
+|===
+|*Type of use case*         |
+|*Description*              |
+|*Material and 3D Structure*|
+|*Influencing Standards*    |
+|*Influencing Technologies* |
+|*Used 3D Model Formats*    |
+|*Used Material Formats*    |
+|*Companies*                |
+|*Relevant User*                 |
+|===
+
+[cols="1,5",caption='Use case 13: ']
+.Using 3D assets linked to material properties in energy/signal propagation to feed sensor (signal processing)  models
+|===
+|*Type of use case*         |
+|*Description*              |
+|*Material and 3D Structure*|
+|*Influencing Standards*    |
+|*Influencing Technologies* |
+|*Used 3D Model Formats*    |
+|*Used Material Formats*    |
+|*Companies*                |
+|*Relevant User*                 |
+|===
+
+
+////
```

Comment:
The commented use-cases can be removed. The topics are already covered by the other use-cases.

Reply:
The commented use-cases still show up in the changes.

Reply:
We've pushed all changes; you should be able to see the correct (current) version now.

Reply:
Now the file is removed, so it is still changed by this PR.

Reply:
Looks good, thanks.